src/browser/provider/built-in/chrome/index.js

```diff
@@ -64,9 +64,19 @@ export default {
     },
 
     async takeScreenshot (browserId, path) {
-        const runtimeInfo = this.openedBrowsers[browserId];
-
-        await cdp.takeScreenshot(path, runtimeInfo);
+        const { client }     = this.openedBrowsers[browserId];
+        const screenshotData = await cdp.getScreenshotData(client);
+        const binaryImage    = Buffer.from(screenshotData.data, 'base64');
+
+        const { visualViewport }            = await client.Page.getLayoutMetrics();
```

Comment:
Please keep all CDP calls in the `cdp` module.

Reply:
BTW we need this cropping logic for videos too, so maybe we should move it to the `cdp.getScreenshotData`?